app.py

```diff
@@ -228,7 +244,7 @@ def update_inspection(inspection_id):  # pragma: no cover
                     )
                 )
                 conn.commit()
-                return inspection.model_dump(), HTTPStatus.OK
+                return inspection.model_dump(indent=2), HTTPStatus.OK
```

Comment:
`indent` doesn't seem to be an argument for that function. Unless we are dealing with a version mismatch here.